src/libraries/System.Data.OleDb/src/OleDb_Util.cs

```diff
@@ -687,6 +687,8 @@ internal static InvalidOperationException IDBInfoNotSupported()
         internal const string DbInfoKeywords = "DbInfoKeywords";
         internal const string Keyword = "Keyword";
 
+        internal static readonly bool IsRunningOnX86 = RuntimeInformation.ProcessArchitecture == Architecture.X86;
```

Comment:
nit: We use _camelCase for internal and private fields. checkout [coding style](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md) for more detail.

Also, since we use this only in OleDbCommand should it move there as private instead?

Reply:
System.Data.OleDb does not follow the coding style. E.g. you can find many other static readonly fields in this file that do not have `s_` prefix.

Reply:
@maryamariyan i expect this flag to be used for other structures as well. Hence I moved it to utilities. Do non private but internal fields also need prefixing? The coding guideline says we should use PascalCase here

Reply:
The coding guidelines say "Prefix internal and private instance fields with `_`, static fields with `s_`"

But I do not think you need to fix this. It is better to use consistent style within given component.